libs/common-const/src/common.ts

```diff
@@ -41,11 +41,39 @@ export const APP_TITLE = 'CoW Swap | The smartest way to trade cryptocurrencies'
 
 type Env = 'barn' | 'prod'
 
-export const COWSWAP_ETHFLOW_CONTRACT_ADDRESS: Record<Env, Record<SupportedChainId, string>> = {
+const NEW_COWSWAP_ETHFLOW_CONTRACT_ADDRESS: Record<Env, string> = {
+  prod: '0xba3cb449bd2b4adddbc894d8697f5170800eadec',
+  barn: '0x04501b9b1d52e67f6862d157e00d13419d2d6e95',
```

Comment:
This should ideally come form the contracts json, right?

Reply:
Not sure, specially during the transicion. I think no package has been published yet, i can make a note for reviewing this point when we deprecate fully the old contracts